.travis.yml

```diff
@@ -2,7 +2,6 @@ sudo: false
 
 env:
   - STACK_YAML=stack.yaml
-  - STACK_YAML=stack-ghc-8.0.2.yaml
```

Comment:
We really want to test on 8.0.2. Please revert.

Reply:
The reason I removed that is because this PR updates servant-auth to use the 9.6 LTS snapshot which uses 8.0.2 (`stack.yaml`). Since that is the case, do we still need this?

Reply:
Ah, I think @jkarni answered my question below. I'll create a `stack-ghc-7.10.3.yaml`.